README.md

```diff
@@ -129,6 +129,23 @@ app.get(
 );
 ```
 
+### initialPage
+
+If you want to change initial Page to sign up, set `mode` or `initialPage`(hosted):
+Values should be matched `login`, `signUp`, and `forgotPassword`
+
+[lock-v11-configuration](https://auth0.com/docs/libraries/lock/v11/configuration#initialscreen-string-)
```

Comment:
I feel this needs a better explanation on how to make it work. Also, it'd be good to note that this only works when using Universal Login Page's in Classic mode. It won't work in the new ULP.

Reply:
@luisrudge We need to use new Universal login. Is there any plan to implements?

Reply:
It's in the backlog, but the ULP team has other priorities right now.

Reply:
@luisrudge thanks for replying, I'll update it after release that.